usagov/vote-gov
# Vote-365: Accessibility Pages Metadata Update

## Jira ticket

[Vote-365](https://cm-jira.usa.gov/browse/VOTE-365)

## Description

This Pr will add metadata to accessibility pages 

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

n/a

### QA/Test

1. Visit [accessibility pages](https://cg-9e8debaf-b030-4825-a43c-cb2bc850c96c.sites.pages.cloud.gov/preview/usagov/vote-gov/feature/vote-365-meta-date-update/accessibility/) and ensure that there are no build errors
2. can also visit https://www.opengraph.xyz/ and paste the build URLs to check that descriptions are present

## Checklist for the Developer




## Checklist for the Peer Reviewers
